test/test_helper.rb

```diff
@@ -1,10 +1,15 @@
 ENV["RAILS_ENV"] ||= "test"
+require "simplecov"
+SimpleCov.start 'rails'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.